cloud-db/acl-overview/ios.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Access Control Overview
+---
+
+[[toc]]
+
+Skygear has provided a default security setting on all records.
+
+By default,
+
+- [read access][doc-read-access] is granted to the public, and
+- [write access][doc-write-access] is granted only to owner of the record.
```

Comment:
This two line should be same as JS/Android.

Reply:
Oh, this link is invalid. Because they are in a tips box, so no anchor.

Maybe I can add link to `ACL Access Level` to the `below` of `More details below`.